admin/admin-service/src/main/java/io/mosip/admin/service/impl/ApplicantDetailServiceImpl.java

```diff
@@ -144,9 +144,15 @@ private DigitalCardStatusResponseDto getDigitialCardStatus(String rid)
             throws Exception {
         List<String> pathsegments=new ArrayList<>();
         pathsegments.add(rid);
-        ResponseWrapper<DigitalCardStatusResponseDto> responseDto = restClient.getApi(ApiName.DIGITAL_CARD_STATUS_URL,pathsegments,"","",ResponseWrapper.class);
+        String response = restClient.getApi(ApiName.DIGITAL_CARD_STATUS_URL,pathsegments,"","",String.class);
+        JSONObject responseObj= objectMapper.readValue(response,JSONObject.class);
```

Comment:
Would like to understand why this change is done and why are handling this as a string?

Reply:
it will not cast directly to response wrapper its saying map is unamed module in bootstrap

Reply:
Resolved